packages/lib-sourcify/src/Validation/fetchUtils.ts

```diff
@@ -0,0 +1,136 @@
+import { logError, logInfo, logDebug } from '../lib/logger';
+import { id as keccak256str } from 'ethers';
+import { IpfsGateway } from '../lib/types';
+
+export async function performFetch(
```

Comment:
Is this only related with Validation? If that's the case, it's ok to keep this here, if it is also used by Verification or other utils let's move it outside from /Validation

Reply:
I don't see this being used anywhere else, except fetching sources, which is part of the validation. If needed we can move it out later.